pkg/cmd/server/origin/master.go

```diff
@@ -377,6 +378,7 @@ func (c *MasterConfig) GetRestStorage() map[string]rest.Storage {
 		RCFn: clientDeploymentInterface{kclient}.GetDeployment,
 		GRFn: deployRollback.GenerateRollback,
 	}
+	configClient, deploymentClient := c.DeploymentConfigControllerClients()
```

Comment:
Looking at usage, either the function name is wrong or the usage here is wrong.  These aren't being used by a controller.